src/use/crypto/certificates.ts

```diff
@@ -0,0 +1,191 @@
+import * as asn1js from "asn1js";
+import Attribute from "pkijs/src/Attribute";
+import CertificationRequest from "pkijs/src/CertificationRequest";
+import { getCrypto } from "pkijs/src/common";
+import Extension from "pkijs/src/Extension";
+import Extensions from "pkijs/src/Extensions";
+import AttributeTypeAndValue from "pkijs/src/AttributeTypeAndValue";
+import { stringToArrayBuffer, arrayBufferToString, toBase64, fromBase64 } from "pvutils";
+import formatPEM from "./formatPem";
+
+const signAlg = "ECDSA";
+const hashAlg = "SHA-256";
+const ellipticCurve = "P-256";
```

Comment:
Am I correct in assuming that is a given by Hyperledger? What if that is ever going to change?

Reply:
Correct. We currently support exaclty the same crypto algorithms as their offical java and node SDKs, if they would change it, we would just need to update the algorithm objects, i.e. change the signing algorithm to RSA or something, remove the EC, ... Everything else should be handled by WebCrypto. 
If they choose to use crypto which is not implemented in WebCrypto, we would need to add some more dependencies which cover that.